### PR TITLE
Support dual-stack

### PR DESCRIPTION
**Pull request by frankui**

### CODE REVIEW

File: Chart.yaml [hunk 1]
```diff
@@ -18,4 +18,4 @@ version: 0.4.10
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 3.20.0
```
Comment (invidian) [inline Chart.yaml:21]: I'd like to keep version 3.20.

File: templates/calico.yaml [hunk 2]
```diff
@@ -664,15 +666,34 @@ spec:
...
+              value: "true"
+            {{- end }}
             # Disable file logging so `kubectl logs` works.
             - name: CALICO_DISABLE_FILE_LOGGING
               value: "true"
             # Set Felix endpoint to host default action to ACCEPT.
             - name: FELIX_DEFAULTENDPOINTTOHOSTACTION
               value: "ACCEPT"
-            # Disable IPv6 on Kubernetes.
+            # Enable IPv6 on Kubernetes.
```
Comment (invidian) [inline templates/calico.yaml:694]: I wonder if IPv6 support should be enabled by default. Is it a non breaking change?

File: values.yaml [hunk 3]
```diff
@@ -7,22 +7,24 @@
...
+podCIDR6: "fd42::/48"
+podIP6BlockSize: 118
 flexVolumePluginDir: "/usr/libexec/kubernetes/kubelet-plugins/volume/exec"
 vethMtu: 0
 autoHostEndpoints: false
 
 typha:
   image:
-    registry: "docker.io"
+    registry: "docker.io/rancher"
```
Comment (invidian) [inline values.yaml:20]: I'd like to use official images when possible.

**Review by invidian — CHANGES_REQUESTED** [review]
Hey @frankui, thanks for your PR. I left few comments, but overall it looks good for merging :) 

We should also bump chart version with those changes.

File: values.yaml (hunk 3 above)
Comment (frankui) [reply values.yaml:20]: I just want to create a PR to enable my RKE2 calico deployement dual-stack in my local repo.  Thanks to look into this and sorry for the bothering. I will close this PR.

**Pull request closed without merging**